release/release.py

```diff
@@ -11,20 +11,21 @@
 
 import git
 
-PART = "part-is-a-mandatory-bump2version-argument-even-when-not-used"
+
+def get_release_folder() -> pathlib.Path:
+    """Return the release folder."""
+    return pathlib.Path(__file__).resolve().parent
 
 
 def get_version() -> str:
     """Return the current version."""
-    command = f"bump2version --list --dry-run --allow-dirty --no-configured-files {PART} .bumpversion.cfg"
-    try:
-        output = subprocess.check_output(command.split(" "), stderr=subprocess.STDOUT, text=True)
-    except (subprocess.CalledProcessError, FileNotFoundError):
-        return "<unknown>"
-    return [line for line in output.split("\n") if line.startswith("current_version")][0].split("=")[1]
+    root = get_release_folder().parent
+    repo = git.Repo(root)
+    latest_tag = sorted(repo.tags, key=lambda tag: tag.commit.committed_datetime)[-1]
```

Comment:
This doesn't look like a safe assumption, perhaps only filter on tag names that resemble a semantic version or those which start with the letter "v"?

Reply:
Yeah, unfortunately bump-my-version has no easy way to return the latest version like bump2version had. I'll add a few more safe guards.